packages/epo-widget-lib/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "esModuleInterop": false,
+    "esModuleInterop": true,
```

Comment:
I disabled this when I was having issues importing epo-react-lib, but reenabled it after I fixed the lib itself and the Jest tests were not liking it off